gapis/api/gles/dependency_graph_behaviour_provider.go

```diff
@@ -323,12 +323,51 @@ func clearBuffer(g *dependencygraph.DependencyGraph, b *dependencygraph.CmdBehav
 	b.Write(g, data)
 }
 
+func (pipeline Pipelineʳ) Uniforms() []Uniform {
+	uniforms := make([]Uniform, 0)
```

Comment:
Apparently, the preferred way is to do `uniforms := []Uniform{}`. I think, only use `make` if specifying an explicit size or capacity. (Also a few times below)